### PR TITLE
Improve test coverage (+3.8%)

### PR DESCRIPTION
**Pull request by kevburnsjr**
I :heart: this package.

### CODE REVIEW

File: parser_test.go [hunk 1]
```diff
@@ -215,6 +216,22 @@ func TestValueInvalidTypeConversion(t *testing.T) {
...
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
+	}
+	_, err = a[2].Uint64()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
+	}
+	_, err = a[2].Float64()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
```
Comment (valyala) [inline parser_test.go]: s/int/float/

File: parser_test.go [hunk 2]
```diff
@@ -215,6 +216,22 @@ func TestValueInvalidTypeConversion(t *testing.T) {
...
+	}
+	_, err = a[2].Uint()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
+	}
+	_, err = a[2].Uint64()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
```
Comment (valyala) [inline parser_test.go]: s/int/uint64/

File: parser_test.go [hunk 3]
```diff
@@ -215,6 +216,22 @@ func TestValueInvalidTypeConversion(t *testing.T) {
 	if err == nil {
 		t.Fatalf("expecting non-nil error when trying to obtain int from string")
 	}
+	_, err = a[2].Int64()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
+	}
+	_, err = a[2].Uint()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
```
Comment (valyala) [inline parser_test.go]: s/int/uint/

File: parser_test.go [hunk 4]
```diff
@@ -215,6 +216,22 @@ func TestValueInvalidTypeConversion(t *testing.T) {
 	if err == nil {
 		t.Fatalf("expecting non-nil error when trying to obtain int from string")
 	}
+	_, err = a[2].Int64()
+	if err == nil {
+		t.Fatalf("expecting non-nil error when trying to obtain int from string")
```
Comment (valyala) [inline parser_test.go]: s/int/int64/

File: parser_test.go (hunk 1 above)
Comment (kevburnsjr) [reply parser_test.go]: Done.

**Comment by valyala** [comment]
Thanks, @kevburnsjr !

**Pull request merged**